### DISCUSSION TITLE
Support macOS Monterey 12

### DISCUSSION THREAD

**Discussion by chris1111** [chris1111/Wireless-USB-Big-Sur-Adapter#101] [General]
Perfect
![aa](https://user-images.githubusercontent.com/6248794/121191081-a1500880-c839-11eb-93f8-12a488f03a57.png)

**Comment by Human79** [comment]
Works for me perfectly.

**Reply by Luka2206** [reply]
Dear Chris , hope you are doing well.  So in 2 words,  for me in the contrary works not perfectly on Monterey 12.6 . I installed the last Wifi driver V13 , my Wifi key is an Archer T2UNANO AC600 . I get a strange behavior , I have a WIFI Time out every 5 or 10 min , then my AC600 lost the signal. Workaround is : Switch Off/On and then It will be work still 10 min .  It means each 10 min I must make this operation. It happens when : - I am 10 mt of distance of my router - or It appears when my laptop is in sleep mode, then I weak up him and AC600 is not recognized , any wifi appears , I cannot select any SSID bus they don't appear . Workaround  Switch Off/On and SSID appears , I can select my SSID and then It will be work . This strange WIFI Time out doesn't appear if I am close my router ( around 1 meter) Somebody can help me ? Thank You in advance

**Comment by rafaelbarahona** [comment]
Hi! i have a problem, I Install the V7 but isnt working after the reboot, the logo keep with the cross on the wifi adaprter

**Reply by Tockman** [reply]
Did you disable SIP?

**Comment by dariushde** [comment]
This doesn't work with Macbook with M1 chip, right? :-(

Thank you

**Reply by chris1111** [reply]
> This doesn't work with Macbook with M1 chip, right? :-(
> 
> Thank you


Exact

**Reply by dariushde** [reply]
Is there any hope to get this to work with the M1 macbooks in the future?

**Reply by faisaljasnak** [reply]
Hi and thank you so much for the work you have done :)

I ran into an issue where i've upgraded my Macbook Air 4,1 to Monterey using Open Core Legacy Patcher. The wireless was working great on High Sierra but after that upgrade i've installed the V13 Big Sur as described but it won't work anymore. I've tried several method but its just not working Neither i'm getting any "Allow" button in Security and Privacy > General Tab as described in the Installer.

My Wifi Dongle is WN725N and I also i've another WN725 but its not working at all it saying NO USB-WIfi Plugin.

**Comment by ToqQrrl** [comment]
Hello Chris ... Will this make the following services work :

1 - The location services ?
2 - Airplay ?
3 - Airdrop ?
4 - Screen Mirroring ?

Thank you in advanced for reading me

**Comment by chris1111** [comment]
@ToqQrrl 
Only AirDrop,  Screen Mirroring not working

**Reply by Tockman** [reply]
In fact, location and Airplay are working well for me.

**Reply by chris1111** [reply]
> In fact, location and Airplay are working well for me.

yes of course I should have mentioned Airdrop and Screen Mirroring

**Reply by JohnShenLee** [reply]
How to get Airdrop to work?

**Comment by ToqQrrl** [comment]
Thank you for the info ... do you guys recommend a particular device ?

**Comment by politbureau** [comment]
Monterey 12.6 here - can't seem to get this to work. Any help appreciated!

Booted into recovery, disabled SIP
Reboot and disable spctl
Open System Pref and ensure "Anywhere" is checked
Install as normal - NOTE that I downloaded V14, but the installer says V12 - not sure if an issue
NOTE - I did not get the "System Extension Updated" pop up, and nothing showing in Security & Privacy Pane
Reboot
Connect Netgear A7000
Icon appears in bar, but indicates "No Wifi USB Plug In"

Any help appreciated!

**Reply by chris1111** [reply]
> Monterey 12.6 here - can't seem to get this to work. Any help appreciated!
> 
> NOTE that I downloaded V14, but the installer says V12 - not sure if an issue
>  NOTE - I did not get the "System Extension Updated" pop up, and nothing showing in Security & Privacy Pane
> Any help appreciated!

Not an issue its a typo fix now
nothing showing in Security & Privacy Pane its probably you have a snapshot disk maybe using OCLP

**Reply by politbureau** [reply]
Thanks for the reply Chris! I realized I installed Monterey on a non-supported Macbook using opencore patcher, so I removed the previous and installed the correct version and it is working 100% now! Instructions worked exactly and I am connected with my Netgear A7000. Thanks for this awesome product - donating now as this saved my bacon! Cheers
